### PR TITLE
changes to showing current port and url

### PR DESCRIPTION
**Pull request by jankrzempek**
What was done:
After changing the url and port now user is able to see current one as a placeholder (backed url settings)

https://trello.com/c/kAzjpy2N

### CODE REVIEW

**Comment by pawelgil** [comment]
Typo in PR title

File: Podfile.lock [hunk 1]
```diff
@@ -1,83 +1,101 @@
 PODS:
-  - Firebase/Analytics (7.5.0):
+  - Firebase/Analytics (8.3.0):
```
Comment (pawelgil) [inline Podfile.lock:2]: this needed? :)

File: AirCasting/Settings/BackendSettingsView.swift [hunk 2]
```diff
@@ -13,20 +13,22 @@ struct BackendSettingsView: View {
     @State private var pathText: String = ""
     @State private var portText: String = ""
     @State private var url: URL?
+    @State private var urlWithoutPort: URL?
+    @State private var port: URL?
```
Comment (pawelgil) [inline AirCasting/Settings/BackendSettingsView.swift:17]: I'm not sure why did you separate those? There can be single URL for this and port can be changed using `URLComponents`:
```Swift
let url = URL(string: "http://www.google.com")!
var components = URLComponents(url: url, resolvingAgainstBaseURL: false)!
components.port = 66
let newUrl = components.url
```

File: AirCasting/URLProviders/UserDefaultsBaseURLProvider.swift [hunk 3]
```diff
@@ -12,6 +12,25 @@ class UserDefaultsBaseURLProvider: BaseURLProvider {
             userDefaults.url(forKey: "baseURL") ?? URL(string: "http://aircasting.org/api")!
         }
     }
+    
+    var baseAppURLwithoutPort: URL {
```
Comment (pawelgil) [inline AirCasting/URLProviders/UserDefaultsBaseURLProvider.swift:16]: like previously - I'm not sure this is needed. Why not single URL with port?

**Review by pawelgil — CHANGES_REQUESTED** [review]

File: Podfile.lock (hunk 1 above)
Comment (jankrzempek) [reply Podfile.lock:2]: 🥶

File: AirCasting/URLProviders/UserDefaultsBaseURLProvider.swift (hunk 3 above)
Comment (jankrzempek) [reply AirCasting/URLProviders/UserDefaultsBaseURLProvider.swift:16]: Now it's obvious, when making I haven't thought about this

**Comment by pawelgil** [comment]
Tests are not passing (not even building) due to some related stuff. We can merge that in, but please take care of it @jankrzempek

**Review by pawelgil — APPROVED** [review]
👌

**Review by smialko — APPROVED** [review]
👏

**Pull request merged**